setup.py

```diff
@@ -7,5 +7,6 @@
     name='coverage_crawler',
     version='1.0.0',
     description='A crawler to find websites that exercise code in Firefox that is not covered by unit tests',
+    install_requires=load_requirements('requirements.txt'),
```

Comment:
You'll also need to add the load_requirements function :)

Reply:
I  just noticed. Sorry for that :)

Reply:
Ah, it looks like setup.py can't handle requirements like the ones starting with "-e", so this simple `load_requirements` is not enough.

You could use the more complex one from https://github.com/mozilla/release-services/blob/13285250a9bdfdac1b4b6c04e4c897dac372d462/src/shipit_code_coverage_backend/setup.py#L15.